_site/leaflet-treering.js

```diff
@@ -2930,7 +2936,13 @@ function CreatePoint(Lt) {
               $(document).keypress(e => {
                 var key = e.which || e.keyCode;
                 if (key === 13) {
-                  Lt.data.year = parseInt(document.getElementById('year_input').value);
+                  if (Lt.measurementOptions.forwardDirection == false && Lt.measurementOptions.subAnnual == false) {
+                    // must subtract one so newest measurment is consistent with measuring forward value
+                    // issue only applies to meauring backwwards annually
+                    Lt.data.year = parseInt(document.getElementById('year_input').value - 1);
```

Comment:
Right now you're subtracting 1 from the value, which will coalesce it to an int anyways, then wrapping all of the in parseInt(). I think you want parseInt(document.getElementById('year_input').value) - 1;